client/src/components/Message/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { keyframes } from '@emotion/core'
+
+const fadeIn = keyframes`
+  0% {
+    display: none;
+    opacity: 0;
+    transform: translateY(50px);
+  }
+
+  100% {
+    display: block;
+    opacity: 1;
+    transform: translateY(0px);
+  }
+`
+
+const Container = styled.div`
+  display: flex;
+  margin: 0px 0px 20px 0px;
+  padding: 20px 0px;
+  animation: ${fadeIn} 0.4s ease-in-out 0.5s;
+`
```

Comment:
This code causes the glitch animation bug.  The **fadeIn** is moving 50px at 0%.  This makes it move immediately to 50px. This is only a problem because you have a delay of *0.4s*.  You need to remove the delay of  *0.4s*.  Then the animation will work properly

See the corrected code below.


```
const Container = styled.div`
  display: flex;
  margin: 0px 0px 20px 0px;
  padding: 20px 0px;
  animation: ${fadeIn} ease-in-out 0.5s;
```

Reply:
Nice one!